build.gradle.kts

```diff
@@ -87,6 +87,10 @@ allprojects {
         }
     }
 
+    dependencies {
+        detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.23.4")
+    }
```

Comment:
Yes, you should. At the moment, there is a discrepancy between the rules applied by ktlint and detekt for styling.
I prefer to drop ktlint and use it via detekt plugin, as above.
https://detekt.dev/docs/rules/formatting/

Reply:
Do tasks like `ktlintFormat` work if used like that? Do you want to propose changes to that plugin, so that we can converge quickly to a shared style?
Also, can you set the ktlint version specifically, or are you bound to the version embedded/used by detekt?

Reply:
I will open a issue on this topic

Reply:
Okay, apparently there's a proposal to drop ktlint from detekt. Too fragile. I prefer to keep them both